lua/diffview/utils.lua

```diff
@@ -23,65 +23,47 @@ function M.now()
   return vim.loop.hrtime() / 1000000
 end
 
----Echo string with multiple lines.
 ---@param msg string|string[]
----@param hl? string Highlight group name.
 ---@param schedule? boolean Schedule the echo call.
-function M.echo_multiln(msg, hl, schedule)
+---@param number level
+function M.notify(msg, level, schedule)
   if schedule then
-    vim.schedule(function() M.echo_multiln(msg, hl, false) end)
+    vim.schedule(function() M.notify(msg, level, false) end)
+    return
+  end
+  if not msg then
+    return
+  end
```

Comment:
Remove this, this function should never be called without a `msg` argument. Unintended usage should not be handled. If anything, do `assert(msg ~= nil)`.